### DISCUSSION TITLE
How to make the power distribution calculated by openmc map onto the mesh of moose?

### DISCUSSION THREAD

**Discussion by LeonDong1998** [neams-th-coe/cardinal#985] [Q&A] [answered]
Hello all！
I found a good result in the initial trial calculation process. The power distribution calculated by OpenMC can be mapped to the mesh.
But after I changed the version recently, I found that this function cannot be realized. The current power distribution requires multiple divisions of the geometry in OpenMC. I want to know how to achieve the mapping in the current version?
![2024-11-10 18](https://github.com/user-attachments/assets/2ec3a07b-183d-4d60-93fe-f0c14cd194a6)

**Comment by aprilnovak** [comment, marked as answer]
Hi @LeonDong1998, I'm not sure exactly what you are asking -- can you share your input files so I can try to better understand?

To me, it looks like you have a mesh tally in the image above - in which case OpenMC treats every element in the `[Mesh]` as a unique tally bin.

**Reply by LeonDong1998** [reply]
Thanks a lot！ 
Based on your description, I have identified the issue. In the latest version of the [Tallies], I should use MeshTally instead of CellTally.

**Reply by LeonDong1998** [reply]
I changed CellTally to MeshTally and the problem was solved. Thank you again！

**Reply by aprilnovak** [reply]
Glad to help!
